### ISSUE TITLE
[bug] Invalid rules for `disable-debug`, `disable-dom`, and `disable-web-apis` configs

### ISSUE THREAD

**Issue by achou11** [Rel1cx/eslint-react#923]
### Describe the bug

Attempting to use `disable-debug`, `disable-dom`, or `disable-web-apis` causes an ESLint error:

```console
Oops! Something went wrong! :(

ESLint: 9.18.0

Configuration for rule "0,off" is invalid. Each rule must have a severity ("off", 0, "warn", 1, "error", or 2) and may be followed by additional options for the rule.

You passed 'undefined', which doesn't contain a valid severity.

If you're attempting to configure rule options, perhaps you meant:

    "0,off": ["error", undefined]

See https://eslint.org/docs/latest/use/configure/rules#using-configuration-files for configuring rules.
```

### Reproduction

1. Follow installation instructions provided by https://eslint-react.xyz/docs/getting-started/typescript
2. Have an ESLint config that is similar to the following:

```ts
// @ts-check
import eslintJs from "@eslint/js";
import eslintReact from "@eslint-react/eslint-plugin";
import tseslint from "typescript-eslint";
 
export default tseslint.config({
  files: ["**/*.ts", "**/*.tsx"],
  extends: [
    eslintJs.configs.recommended,
    tseslint.configs.recommended,
    eslintReact.configs.recommended,
    eslintReact.configs['disable-debug'], // Add this line!
  ],
  languageOptions: {
    parser: tseslint.parser,
    parserOptions: {
      projectService: true,
    },
  },
});
```

3. Run ESLint e.g. `npx eslint --cache .`

### Expected behavior

Applying the config(s) should work and not cause ESLint to fail.

### Platform and versions

```Text
Node@20.18.0
@eslint-react/eslint-plugin@1.24.0
```

### Stack trace

```Text

```

### Additional context

When logging something like `eslintReact.configs['disable-debug']`, I get `{ '0,off': undefined }`
